src/test/functional/com/hltech/pact/gen/testfeignclient/TestDto.groovy

```diff
@@ -0,0 +1,114 @@
+package com.hltech.pact.gen.testfeignclient
+
+class TestDto {
```

Comment:
Maybe we can add generic and non-optional field in this DTO? Or maybe make this DTO generic?